sentry_sdk/integrations/_wsgi_common.py

```diff
@@ -124,9 +123,12 @@ def parsed_body(self):
         form = self.form()
         files = self.files()
         if form or files:
-            data = dict(iteritems(form))
-            for key, _ in iteritems(files):
-                data[key] = AnnotatedValue.removed_because_raw_data()
+            data = {}
+            if form:
+                data = dict(form.items())
+            if files:
+                for key in files.keys():
+                    data[key] = AnnotatedValue.removed_because_raw_data()
```

Comment:
```suggestion
            data = None
            if form:
                data = dict(form.items())
            if files:
                data = {}
                for key in files.keys():
                    data[key] = AnnotatedValue.removed_because_raw_data()
```

See also my comment below